### PR TITLE
added moneydoor implementation

### PR DESCRIPTION
**Pull request by aliuliu121**
accept me

### CODE REVIEW

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
+package edu.cmu.cs.cs214.rec10.plugin;
+
+import edu.cmu.cs.cs214.rec10.framework.core.GameFramework;
+import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
+import edu.cmu.cs.cs214.rec10.framework.core.Player;
+
+import java.util.*;
```
Comment (coffeeologist) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java:7]: * imports are baaaaaaaaaaaaaaaaaaaaaaaaaaad

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+public class MoneyDoorPlugin implements GamePlugin {
+
+    private int[] arr = {-3, -2, -1, 0, 1, 2, 3, 4, 5, 6};
```
Comment (coffeeologist) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java:11]: Hard coded arrays??!??!?!

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        return true;
+    }
+
+    public String getGameOverMessage() {
+        if (PlayerScores.get("X") > PlayerScores.get("O")) {
+            return "Congratulations X with a score of " + PlayerScores.get("X") + " !";
+        } else {
+            return "Congratulations O with a score of " + PlayerScores.get("O") + " !";
+        }
+
```
Comment (coffeeologist) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java]: unacceptable spacing >:3

**Review by coffeeologist — CHANGES_REQUESTED** [review]
hmph

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java (hunk 1 above)
Comment (aliuliu121) [reply src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java:7]: no u

**Review by coffeeologist — APPROVED** [review]

**Comment by coffeeologist** [comment]
good job uwu

**Pull request merged**